### PR TITLE
RHDEVDOCS-3934 - 5.3.5  RNs

### PR DESCRIPTION
**Pull request by libander**
Aligned team: Dev Tools
For branches: 4.9+
Jira:https://issues.redhat.com/browse/RHDEVDOCS-3934
Direct link to doc preview:
SME review: @alanconway
QE review: @anpingli
Peer review: @bburt-rh

### CODE REVIEW

File: modules/cluster-logging-release-notes-5.3.z.adoc [hunk 1]
```diff
@@ -1,5 +1,29 @@
 //Z-stream Release Notes by Version
 
+[id="cluster-logging-release-notes-5-3-5"]
+= OpenShift Logging 5.3.5
+This release includes link:https://access.redhat.com/errata/RHSA-2022:0721[RHSA-2022:0721 OpenShift Logging Bug Fix Release 5.3.5]
```
Comment (bburt-rh) [inline modules/cluster-logging-release-notes-5.3.z.adoc:5]:
```suggestion
[role="_abstract"]
This release includes link:https://access.redhat.com/errata/RHSA-2022:0721[RHSA-2022:0721 OpenShift Logging Bug Fix Release 5.3.5]
```
I think you need to add a line before the first sentence to mark it as an abstract

File: modules/cluster-logging-release-notes-5.3.z.adoc [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+
+[id="openshift-logging-5-3-5-bug-fixes"]
+== Bug fixes
+* Before this update, if you removed OpenShift Logging from {product-title}, the web console continued displaying a link to the *Logging* page. With this update, removing or uninstalling OpenShift Logging also removes that link. (link:https://issues.redhat.com/browse/LOG-2182[LOG-2182])
```
Comment (bburt-rh) [inline modules/cluster-logging-release-notes-5.3.z.adoc:9]: Should `OpenShift Logging` be replaced with an attribute?

Comment (libander) [reply modules/cluster-logging-release-notes-5.3.z.adoc:9]: No, this is for 5.3, attributes were added for the name change that goes with 5.4

File: modules/cluster-logging-release-notes-5.3.z.adoc (hunk 1 above)
Comment (libander) [reply modules/cluster-logging-release-notes-5.3.z.adoc:6]: Fixed, thanks!

**Pull request closed without merging**